composer.json

```diff
@@ -36,10 +37,14 @@
     },
     "autoload": {
         "psr-0": {
+            "TYPO3\\Flow\\Log\\": "Neos.Flow.Log/Classes",
             "TYPO3\\Eel": "TYPO3.Eel/Classes",
             "TYPO3\\Flow": "TYPO3.Flow/Classes",
             "TYPO3\\Fluid": "TYPO3.Fluid/Classes",
             "TYPO3\\Kickstart": "TYPO3.Kickstart/Classes"
+        },
+        "psr-4": {
+            "Neos\\Flow\\Log\\": "Neos.Flow.Log/Classes"
```

Comment:
Sorry if I'm thick, but I just *had* a WTF moment ;)
So you want to create dummy classes at the PSR-4 location and keep the PSR-0 ones as well??